Specs/DateTime/English/DateTimeModel.json

```diff
@@ -20888,5 +20888,125 @@
         }
       }
     ]
+  },
+  {
+    "Input": "They worked on the problem for the last 3 months and 2 weeks and 2 days ago they found a solution",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T16:12:00"
+    },
+    "NotSupportedByDesign": "javascript,python,java",
```

Comment:
Add a Comment attribute here describing that the middle entity is not merged due to ambiguity.

Reply:
Same in the equivalent cases in other languages.

Reply:
added comment (and test case in Italian and Spanish)